BTVN10/main.js

```diff
@@ -0,0 +1,167 @@
+//b1:in bang nhan
+const getMultiply = (number) => {
+    console.log('In bang nhan:')
+    for(let i=1; i<=10; i++){
+        console.log(number, 'x', i, '=', number*i)
+    }
+}
+//b2:in cac so chan tu 1 den n
+const checkEven = (n) => {
+    if(n<1 || n>30) console.log('Nhap lai n')
```

Comment:
indent là 2 dấu cách (1 tab là 2 dấu cách)
Và nhìn kỹ cách viết dấu cách như sau:
```suggestion
    if (n < 1 || n > 30) {
      console.log('Nhap lai n')
    }
```